### ISSUE TITLE
Doesn't work with Samsung A3 2017 SM-A300FU ?

### ISSUE THREAD

**Issue by Qu1goN** [0x192/Universal-Android-Debloater#596]
It doesn't work with my A3 - did not find my Phone?

**Comment by AnonymousWP** [comment]
Are you sure you followed [the instructions](https://github.com/0x192/universal-android-debloater#how-to-use-it)?

**Comment by Qu1goN** [comment]
Yes, I just tried it again and suddenly the connection works ^^ But only a few apps are listed and only if I select unlisted instead of recommended... how do I get the apps that cannot be uninstalled uninstalled and how do I know what what is? It's not just the app name there ^^
![Screen003](https://user-images.githubusercontent.com/62997079/219869682-fdbd7f0d-70e8-48ea-89be-bbd5a6de8d27.jpg)

**Comment by AnonymousWP** [comment]
Which apps are unlisted? If apps can't be uninstalled, you can disable them. To do that, go to settings (top-right) and check the box about disabling.

**Comment by Qu1goN** [comment]
The preinstalled apps are the problem. The A3 only has 16GB anyway and deactivating it doesn't give me more storage space. :(

**Comment by AnonymousWP** [comment]
> The preinstalled apps are the problem. The A3 only has 16GB anyway and deactivating it doesn't give me more storage space. :(

Yes, so what are the package names? Maybe you can try ctrl + f [here](https://github.com/0x192/universal-android-debloater/blob/main/resources/assets/uad_lists.json) and check whether the packages are present or not.

**Comment by Qu1goN** [comment]
The few entries on the screenshot above is all that is displayed by the program. No matter what I select in the dropdown menus at the top of the program. Is it at all possible with your program to uninstall things without root? E.g. Flipboard, Currents, Drive, Email, Galaxy Store, Google TV, Hangouts, Chrome, Play Music etc.
I've already tried to root the smartphone but it didn't work or I couldn't do it.
Samsung Galaxy A3 SM-A300FU - Android 6.0.1 - Build MMB29M.A300FUXXS1CQD5

**Comment by Qu1goN** [comment]
I rootet my A3 - tryed another USB-Port and it works but ihave only the option to deaktiviate not to deinstall the Apps?

**Comment by AnonymousWP** [comment]
Uninstalling isn't possible without root, see https://github.com/0x192/universal-android-debloater/wiki/FAQ#how-is-it-possible-to-delete-a-system-package-without-root-permission. You can just disable them. Disabling is just as fine; it stops the process and its data is wiped. The app is still present, but inactive.
